packages/eslint-plugin-pf-codemods/lib/rules/v5/resizeObserver-function-param.js

```diff
@@ -0,0 +1,53 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8324
+module.exports = {
+  meta: { hasSuggestions: true },
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-core").filter(
+      (specifier) => specifier.imported.name === "getResizeObserver"
+    );
+
+    return imports.length == 0
+      ? {}
+      : {
+          CallExpression(node) {
+            if (
+              node.callee.name === "getResizeObserver" &&
+              node.arguments.length !== 3
+            ) {
+              context.report({
+                node,
+                message: `A third parameter, useRequestAnimationFrame, has been added to the getResizeObserver function.`,
+                suggest: [
+                  {
+                    desc: "Pass in a value of false when the callback passed in is debounced. This maintains the previous functionality.",
+                    fix: function (fixer) {
+                      if (node.arguments.length !== 3) {
+                        const lastToken = context.getLastToken(node);
+                        const fixes = [
+                          fixer.insertTextBefore(lastToken, ", false"),
+                        ];
+                        return fixes;
+                      }
+                    },
+                  },
+                  {
+                    desc: "Omit the third argument or pass in a value of true if the callback passed in is not debounced. This is the new default functionality.",
```

Comment:
Wouldn't omitting the 3rd argument be the same as passing false?

![image](https://user-images.githubusercontent.com/5322142/210437447-f28b1ce9-8c3c-43e7-aa50-28462ba028de.png)

Reply:
Good catch! I got the functionality on the v5 branch mixed up with the v4 functionality I think.